user_guide_src/source/general/errors.rst

```diff
@@ -131,3 +131,32 @@ forcing a redirect to a specific route or URL::
 redirect code to use instead of the default (``302``, "temporary redirect")::
 
     throw new \CodeIgniter\Router\Exceptions\RedirectException($route, 301);
+
+
+Custom Exception Handler
+------------------------
```

Comment:
`Custom Exceptions` indicates CodeIgniter4 custom exceptions for app developers.
This naming may be not good. 

In the other hand, `Custom Exception Handler` means handling app developers custom exceptions.
It should not be included in `Custom Exceptions` section.

```suggestion
========================
```